### ISSUE TITLE
Confusing `start-date` deprecation

### ISSUE THREAD

**Issue by rzetelskik** [scylladb/scylla-manager#3701]
[sctool claims that `start-date` is deprecated](https://github.com/scylladb/scylla-manager/blob/d96e5d8338dd0afa5e130ee8cbb88b9cb11bc94b/pkg/command/flag/task.yaml#L39), yet there is no mention of it in its [swagger specification](https://github.com/scylladb/scylla-manager/blob/d96e5d8338dd0afa5e130ee8cbb88b9cb11bc94b/swagger/scylla-manager.json#L276), nor the [generated model](https://github.com/scylladb/scylla-manager/blob/d96e5d8338dd0afa5e130ee8cbb88b9cb11bc94b/swagger/gen/scylla-manager/models/schedule.go#L34).
Is this only a deprecated flag in sctool, or should a user refrain from using it with the REST API as well? If the latter is the case, it should be documented.

cc @Michal-Leszczynski

**Comment by karol-kokoszka** [comment]
According to the documentation start-date is deprecated indeed. 
https://manager.docs.scylladb.com/stable/sctool/backup.html#s-start-date

The point is that `start-date` is not fully replaceable with the cron expression...
Cron expression defines the frequency, so it's not possible to define when to start.  
`cron` should deprecate the `interval` flag only.

I'll make a change in the documentation to remove `start-date` deprecation and will update swagger to deprecate just the interval.

**Comment by rzetelskik** [comment]
> The point is that start-date is not fully replaceable with the cron expression...

Agreed, I logged it in https://github.com/scylladb/scylla-manager/issues/3700.

> cron should deprecate the interval flag only.

Bare in mind the other issue https://github.com/scylladb/scylla-manager/issues/3702. A broader solution could cover all three issues. We discussed with @Michal-Leszczynski that the standard cron syntax could possibly be extended to cover this case and parsed on the server side entirely (as opposed to `start-date` having some special cases parsed on the client side). Not saying this is the right approach, but with the issues related to start-date syntax, keeping the deprecation might be worth considering.
